docs/en/sql-reference/statements/truncate.md

```diff
@@ -25,17 +27,33 @@ You can specify how long (in seconds) to wait for inactive replicas to execute `
 If the `alter_sync` is set to `2` and some replicas are not active for more than the time, specified by the `replication_wait_for_inactive_replica_timeout` setting, then an exception `UNFINISHED` is thrown.
 :::
 
+The `TRUNCATE TABLE` query is **not supported** for the following table engines:
+
+- [View](../../engines/table-engines/special/view.md)
+- [File](../../engines/table-engines/special/file.md)
+- [URL](../../engines/table-engines/special/url.md)
+- [Buffer](../../engines/table-engines/special/buffer.md)
+- [Null](../../engines/table-engines/special/null.md)
```

Comment:
```suggestion
- [`View`](../../engines/table-engines/special/view.md)
- [`File`](../../engines/table-engines/special/file.md)
- [`URL`](../../engines/table-engines/special/url.md)
- [`Buffer`](../../engines/table-engines/special/buffer.md)
- [`Null`](../../engines/table-engines/special/null.md)
```

I've also been back-ticking function names. Not really opinionated on this either, as long as we apply it consistently.